src/example/stokes_flow/stokes_flow.jl

```diff
@@ -0,0 +1,340 @@
+#   \nabla \cdot u = 0 \textrm{  in } \Omega  \\
+#   u = (0,0)  \textrm{  on } \Gamma_L \cup \Gamma_R \\
+#   u = (1,0) \textrm{ and } p=0 \textrm{  on } \Gamma_T
+# \end{cases}
+# ```
+# Density $\rho$ is taken equal to $1400kg.m^{-3}$ and dynamic viscosity $\mu$ equal to $1.4 Pa.s$. The Reynolds number based on the height of the wedge is therefore equal to $1000$.
+# The weak form of this problem is given by:
+# ```math
+# \begin{cases}
+#   \textrm{ Find } u \in V \textrm{ and } p \in Q \textrm{ such that } \forall v \in V, \, \forall q \in Q\\
+#   \int_\Omega \left( \mu \nabla u : \nabla v - p \nabla \cdot v \right) \, dx  = \int_\Omega f \cdot v \, dx  \\
+#   \int_\Omega q \nabla \cdot u\, dx  = 0  
+# \end{cases}
+# ```
+# where $V$ and $Q$ are suitable function spaces. This can be recast in "mixed" formalism:
+# ```math
+# \begin{cases}
+#   \textrm{ Find } (u,p) \in W=V \times Q  \textrm{ such that } \forall (v,q) \in W\\
+#   a((u,p),(v,q)) = l((v,q))
+# \end{cases}
+# ``` 
+# where $a((u,p),(v,q)) = \int_\Omega \left( \mu \nabla u : \nabla v - p \nabla \cdot v +q \nabla \cdot u \right)\, dx$ and
+# $l((v,q))=\int_\Omega f \cdot v \, dx$
+# 
+# Taylor-Hood ($\mathbb{P2}-\mathbb{P1}$) elements are used to discretize the weak form of the problem which leads to the linear system:
+# ```math
+# \begin{bmatrix}
+#  A & B^T \\
+#  B & 0 
+# \end{bmatrix} 
+# \begin{bmatrix} 
+# U \\
+# P
+# \end{bmatrix}
+# = L
+# ``` 
+#
+# ## Commented code
+
+# import necessary packages
+using Bcube
+using LinearAlgebra
+using WriteVTK
+using StaticArrays
+using SpecialFunctions
+
+# Function space: P2 for velocity and P1 for pressure
+const fspace = :Lagrange
+const degree_u = 2
+const degquad = 2 * degree_u + 1
+const degree_p = 1
+
+# Material parameters
+const ρ = 1400.0
+const μ = 14.0
+const ν = μ / ρ
+
+# Parameters for the oscillating plate case
+const f = 1.0
+const k = sqrt(π * f / ν)
+
+const Δt = 1.0e-3
+const finalTime = 2.0
+
+const outputpath = joinpath(@__DIR__, "..", "..", "..", "myout", "stokes")
+mkpath(outputpath)
+
+# Function that runs the steady case:
+function run_steady()
+    # Read 2D mesh
+    mesh_path =
+        joinpath(@__DIR__, "..", "..", "..", "input", "mesh", "domainTriangle_tri.msh")
+    mesh = read_msh(mesh_path)
+
+    fsu = FunctionSpace(fspace, degree_u)
+    U_vel = TrialFESpace(
+        fsu,
+        mesh,
+        Dict(
+            "edge_right" => SA[0.0, 0.0],
+            "edge_left" => SA[0.0, 0.0],
+            "edge_top" => SA[1.0, 0.0],
+        );
+        size = 2,
+    )
+    V_vel = TestFESpace(U_vel)
+
+    fsp = FunctionSpace(fspace, degree_p)
+    U_pre = TrialFESpace(fsp, mesh, Dict("edge_top" => SA[0.0]); size = 1)
+    V_pre = TestFESpace(U_pre)
+
+    # Define MultiFESpace (mixed formalism)
+    U = MultiFESpace(U_vel, U_pre)
+    V = MultiFESpace(V_vel, V_pre)
+
+    velocity = FEFunction(U_vel, 0.0)
```

Comment:
`velocity = FEFunction(U_vel)` initializes with zeros. But if you prefer a more explicit init, you can leave it as is.